app/controllers/proofs_controller.rb

```diff
@@ -36,7 +36,7 @@ def initialize_api_config
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def resource_params
-    params.require(:proof).permit(:proof_type_id, :identity_id, :encryption_fingerprint, files: [])
+    params.require(:proof).permit(:proof_type_id, :entity_id, :encryption_fingerprint, :entity, files: [])
```

Comment:
why not `entity_type` instead of `entity` ?

`entity` name looks more like object, not field